build/gtsrb/build.py

```diff
@@ -28,20 +28,38 @@
 
 import finn.builder.build_dataflow as build
 import finn.builder.build_dataflow_config as build_cfg
+from finn.util.basic import alveo_default_platform
 from finn.builder.build_dataflow_config import default_build_dataflow_steps
+from qonnx.transformation.insert_topk import InsertTopK
+from finn.builder.build_dataflow_steps import build_dataflow_step_lookup
 from qonnx.core.datatype import DataType
+from qonnx.core.modelwrapper import ModelWrapper
 import os
 import shutil
 import numpy as np
+import onnx
 from onnx import helper as oh
 
-models = [
-    "cnv_1w1a_gtsrb",
-]
+
+model_name = "cnv_1w1a_gtsrb"
+model_file = "models/%s.onnx" % model_name
+
+verif_en = os.getenv("VERIFICATION_EN", "0")
 
 # which platforms to build the networks for
 zynq_platforms = ["Pynq-Z1"]
-platforms_to_build = zynq_platforms
+alveo_platforms = []
+platforms_to_build = zynq_platforms + alveo_platforms
+
+
+def custom_step_update_model(model, cfg):
```

Comment:
Maybe we can move this function to a central point, since it is used in multiple places. Either to finn or to qonnx, we can sync offline about this.